### PR TITLE
Update VirtualMachineCRCErrors runbook to handle aggregated alert

### PR DESCRIPTION
**Pull request by machadovilaca**
This PR updates `VirtualMachineCRCErrors` runbook to reflect that the alert now aggregates all VMs with issues, so we are adding a script so that users can more easily find which VMs are subject to problems

### CODE REVIEW

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 1]
```diff
@@ -14,26 +14,54 @@ major service outages.
...
-```
-
-3) Get the storage class configuration:
-
-```bash
-$ kubectl get sc <storage-class-name> -o yaml
-```
-
-4) Check if the storage class has the "krbd:rxbounce" map option:
-
```
Comment (apinnick) [inline docs/runbooks/VirtualMachineCRCErrors.md:36]: An elipsis in a YAML file should be `# ...`, not `...`.

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 2]
```diff
@@ -14,26 +14,54 @@ major service outages.
 
 ## Diagnosis
 
-1) Get the volume name from the VM:
+Execute the following bash script to get a list of Virtual Machines that are
+using a storage class that is incorrectly configured:
 
 ```bash
-$ kubectl get vm <vm-name> -o jsonpath='{.spec.template.spec.volumes}'
+#!/bin/bash
```
Comment (apinnick) [inline docs/runbooks/VirtualMachineCRCErrors.md:21]: Are we really going to require downstream users to run a script to check storage classes? Is this the only solution?

Comment (machadovilaca) [reply docs/runbooks/VirtualMachineCRCErrors.md:21]: I think it is possible to ask them to query the `kubevirt_ssp_vm_rbd_volume == 1` metric, and should output the VMs with issues and the PV name

Comment (apinnick) [reply docs/runbooks/VirtualMachineCRCErrors.md:21]: If you can write the query, I can make sure that it is in the downstream docs. I think there is time, so no rush.

File: docs/runbooks/VirtualMachineCRCErrors.md (hunk 1 above)
Comment (machadovilaca) [reply docs/runbooks/VirtualMachineCRCErrors.md:36]: updated

File: docs/runbooks/VirtualMachineCRCErrors.md (hunk 2 above)
Comment (machadovilaca) [reply docs/runbooks/VirtualMachineCRCErrors.md]: please take a new look

Comment (apinnick) [reply docs/runbooks/VirtualMachineCRCErrors.md]: LGTM

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 3]
```diff
@@ -14,26 +14,28 @@ major service outages.
 
 ## Diagnosis
 
-1) Get the volume name from the VM:
+Execute the following PromQL expression to get a list of Virtual Machines that
```
Comment (apinnick) [inline docs/runbooks/VirtualMachineCRCErrors.md:17]: Suggestion: "Obtain a list of VirtualMachines with an incorrectly configured storage class by running the following PromQL query:"

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 4]
```diff
@@ -14,26 +14,28 @@ major service outages.
...
 
-2) Get the storage class name from the volume:
+|                            | name               | namespace | pv_name            | rxbounce_enabled | volume_mode | value |
+|----------------------------|--------------------|-----------|--------------------|------------------|-------------|-------|
+| kubevirt_ssp_vm_rbd_volume | testvmi-gwgdqp22k7 | test-ns   | testvmi-gwgdqp22k7 | false            | Block       | 1     |
 
-```bash
-$ kubectl get pvc <volume-name> -o jsonpath='{.spec.storageClassName}'
-```
+You will see a list of Virtual Machines that are using a storage class without
```
Comment (apinnick) [inline docs/runbooks/VirtualMachineCRCErrors.md:30]: Suggestion: "The output displays a list of VirtualMachines that use a storage class without `rxbounce_enabled`.

File: docs/runbooks/VirtualMachineCRCErrors.md [hunk 5]
```diff
@@ -14,26 +14,28 @@ major service outages.
...
+`rxbounce_enabled`.
 
-3) Get the storage class configuration:
+To get the storage class name, execute the following:
```
Comment (apinnick) [inline docs/runbooks/VirtualMachineCRCErrors.md:33]: Suggestion: "Obtain the storage class name by running the following command:"

**Comment by machadovilaca** [comment]
@apinnick all updated

**Comment by apinnick** [comment]
> @apinnick all updated

LGTM

**Review by sradco — APPROVED** [review]

**Pull request merged**